src/registry_schemas/schemas/alteration.json

```diff
@@ -1,15 +1,16 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/alteration",
-  "anyOf": [
```

Comment:
don't we still need at least 1 change for an alteration?

Reply:
Yes but the business legal type is always required for a change right? That's why I added the 'business' as required further down -- the other changes are only allowed if the business legal type is also changing

Reply:
otherwise you could have an alteration that only included a name change or something